modules/pytket-qiskit/pytket/extensions/qiskit/qiskit_convert.py

```diff
@@ -446,13 +455,15 @@ def append_tk_command_to_qiskit(
 
     # others are direct translations
     try:
-        gatetype = _known_qiskit_gate_rev[optype]
+        gatetype, phase = _known_gate_rev_phase[optype]
     except KeyError as error:
         raise NotImplementedError(
             "Cannot convert tket Op to Qiskit gate: " + op.get_name()
         ) from error
     params = [param_to_qiskit(p, symb_map) for p in op.params]
     g = gatetype(*params)
+    if phase != 0.0:
+        qcirc.global_phase += phase * sympy.pi
```

Comment:
I guess making this conditional is a little optimization but stylistically I think it would be nicer to do the addition unconditionally.